diff -uNr zypper-1.6.10/src/CMakeLists.txt zypper-1.6.10.new/src/CMakeLists.txt
--- zypper-1.6.10/src/CMakeLists.txt	2011-06-09 16:49:08.000000000 +0900
+++ zypper-1.6.10.new/src/CMakeLists.txt	2011-08-12 11:41:31.275372002 +0900
@@ -92,10 +92,10 @@
 )
 
 ADD_LIBRARY( zypper_lib ${zypper_SRCS} ${zypper_out_SRCS} ${zypper_utils_SRCS} )
-TARGET_LINK_LIBRARIES( zypper_lib ${ZYPP_LIBRARY} ${READLINE_LIBRARY} -laugeas ${AUGEAS_LIBRARY} )
+TARGET_LINK_LIBRARIES( zypper_lib ${ZYPP_LIBRARY} ${READLINE_LIBRARY} -laugeas -lncurses ${AUGEAS_LIBRARY} )
 
 ADD_EXECUTABLE( zypper main.cc )
-TARGET_LINK_LIBRARIES( zypper zypper_lib ${ZYPP_LIBRARY} ${READLINE_LIBRARY} -laugeas ${AUGEAS_LIBRARY} -lrt )
+TARGET_LINK_LIBRARIES( zypper zypper_lib ${ZYPP_LIBRARY} ${READLINE_LIBRARY} -laugeas -lncurses ${AUGEAS_LIBRARY} -lrt )
 
 
 INSTALL(
